src/DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('server')->cannotBeEmpty()->end()
                         ->scalarNode('enabled')->defaultFalse()->end()
                         ->arrayNode('ip_white_list')
+                            ->setDeprecated('The "%node%" option is deprecated. Use "trusted_ip_list" instead with the same values.')
+                        ->end()
+                        ->arrayNode('trusted_ip_list')
```

Comment:
It would be great to throw an exception if both configuration nodes are defined. I'm not sure if it can be done here, if yes, that's great, if not you can do it in the extension.